preferences.yaml

```diff
@@ -0,0 +1,4 @@
+timetolive:
```

Comment:
let's not commit this file. i added it to .gitignore in https://github.com/USGS-VIZLAB/hurricane-maria/pull/18, a minute too late for you. but can you just undo this file?